### PR TITLE
feat: Styling issues with the asm ui expand collapse arrow (GH-5335)-2.0

### LINKED ISSUE #5335: Styling issues with the asm ui expand collapse arrow
**Environment Details**
Spartacus: develop branch
Device: mac
Happens on: http://dev-com-10.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD/?asm=true

**Styling Issues**
- On mobile viewport, you can see the _expand/collapse arrow_ is on top of the list. It should be hidden behind the list. 
Screen Shot 2019-11-07 at 1 53 57 PM

- On tablet viewport, you can see the _expand/collapse arrow_ is overlaid on the _Search input field_. 
Screen Shot 2019-11-07 at 1 55 00 PM

Screen Shot 2019-11-07 at 1 55 07 PM

- On firefox, you can see the _expand/collapse arrow_ isn't quite aligned within its box. 
Screen Shot 2019-11-07 at 1 56 03 PM

Screen Shot 2019-11-07 at 2 13 18 PM

### PR DESCRIPTION
**Pull request by bgambocjaviniar**
closes GH-5335

Note:
- style in component level 
-->It is here as it is still in other asm component level. Miguel said it was decided to be left until further notice, however, there's an improvement ticket for it in the future https://github.com/SAP/spartacus/issues/5513

- svg fill value is the same as the others (only logout fill value is diff from the rest)

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts [hunk 1]
```diff
@@ -39,7 +44,8 @@ export class AsmMainUiComponent implements OnInit {
     protected userService: UserService,
     protected asmComponentService: AsmComponentService,
     protected globalMessageService: GlobalMessageService,
-    protected routingService: RoutingService
+    protected routingService: RoutingService,
+    protected asmService: AsmService
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts:45]: Should we add this to schematics?

File: projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
+cx-asm-toggle-ui {
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss:1]: Please move to styles library

File: projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts [hunk 3]
```diff
@@ -55,6 +61,11 @@ export class AsmMainUiComponent implements OnInit {
         }
       })
     );
+    this.subscription.add(
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts]: Can we put this in an async pipe instead?

File: projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    .collapseIcon,
+    .expandIcon {
+      width: 16px;
+      height: 16px;
+    }
+
+    .collapseIcon {
+      background: url("data:image/svg+xml,%3Csvg aria-hidden='true' focusable='false' data-prefix='fas' data-icon='angle-double-up' class='svg-inline--fa fa-angle-double-up fa-w-10' role='img' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 320 512'%3E%3Cpath fill='%23d1e3ff' d='M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z'%3E%3C/path%3E%3C/svg%3E")
+        center center no-repeat;
+      background-color: rgba(0, 0, 0, 0.5);
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss]: Please double check the color

File: projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import { Component, OnDestroy, OnInit, ViewEncapsulation } from '@angular/core';
+import { AsmService } from '@spartacus/core';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-asm-toggle-ui',
+  templateUrl: './asm-toggle-ui.component.html',
+  styleUrls: ['./asm-toggle-ui.component.scss'],
+  encapsulation: ViewEncapsulation.None,
```
Comment (LTiger14) [inline projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.ts:9]: Why do you need this?

**Review by LTiger14 — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss (hunk 2 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss:1]: will include in the styling moving ticket

File: projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss (hunk 4 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/asm/asm-toggle-ui/asm-toggle-ui.component.scss]: will remove

File: projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts (hunk 1 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts:45]: will do

File: projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts (hunk 3 above)
Comment (bgambocjaviniar) [reply projects/storefrontlib/src/cms-components/asm/asm-main-ui/asm-main-ui.component.ts]: fixed

File: projects/schematics/src/migrations/2_0/constructor-deprecations/data/asm-main-ui.component.migration.ts [hunk 6]
```diff
@@ -0,0 +1,51 @@
+import {
```
Comment (LTiger14) [inline projects/schematics/src/migrations/2_0/constructor-deprecations/data/asm-main-ui.component.migration.ts]: My bad, this component isn't exported

**Review by LTiger14 — CHANGES_REQUESTED** [review]

**Review by LTiger14 — APPROVED** [review]

**Review by znikola — APPROVED** [review]

**Pull request merged**